### ISSUE TITLE
Pull request schema parse error

### ISSUE THREAD

**Issue by Hagith** [git-town/action#46]
Hello, I started experiencing an issue with git-town action. Here is one of the actions failed: https://github.com/coldrun/monorepo-typescript/actions/runs/12139006387/job/33845664491?pr=23

It is quite strange as pull request schema has `body` marked as `string().optional(),`, but for given example fails with error:
```json
{
  "code": "invalid_type",
  "expected": "string",
  "received": "null",
  "path": [
    "body"
  ],
  "message": "Expected string, received null"
}
```

**Comment by stephenwade** [comment]
`string().optional()` makes the type `string | undefined`. I guess we also need `.nullable()`.

**Comment by tranhl** [comment]
@Hagith Thanks for taking the time PR the fix! This has been release with [v1.0.7](https://github.com/git-town/action/releases/tag/v1.0.7).
